tests/test_utils/test_schemaview.py

```diff
@@ -1173,56 +1223,61 @@ def test_schemaview(schema_view_no_imports: SchemaView) -> None:
     assert set(view.get_mappings(ACTIVITY)["exact"]) == {"prov:Activity"}
     assert set(view.get_mappings(ACTIVITY, expand=True)["exact"]) == {"http://www.w3.org/ns/prov#Activity"}
 
+
+def test_schema_usage(schema_view_no_imports: SchemaView) -> None:
+    """Test usage_index method and SchemaUsage objects."""
+    view = schema_view_no_imports
     u = view.usage_index()
     for k, v in u.items():
         logger.debug(f" {k} = {v}")
 
-        assert (
-            SchemaUsage(
-                used_by="FamilialRelationship", slot=RELATED_TO, metaslot="range", used="Person", inferred=False
-            )
-            in u["Person"]
-        )
-        assert [
-            SchemaUsage(
-                used_by="Person",
-                slot="reason_for_happiness",
-                metaslot="any_of[range]",
-                used="MarriageEvent",
-                inferred=True,
-            ),
-            SchemaUsage(
-                used_by="Adult",
-                slot="reason_for_happiness",
-                metaslot="any_of[range]",
-                used="MarriageEvent",
-                inferred=False,
-            ),
-        ] == u["MarriageEvent"]
-        assert [
-            SchemaUsage(
-                used_by="Person", slot="has employment history", metaslot="range", used="EmploymentEvent", inferred=True
-            ),
-            SchemaUsage(
-                used_by="Person",
-                slot="reason_for_happiness",
-                metaslot="any_of[range]",
-                used="EmploymentEvent",
-                inferred=True,
-            ),
-            SchemaUsage(
-                used_by="Adult", slot="has employment history", metaslot="range", used="EmploymentEvent", inferred=False
-            ),
-            SchemaUsage(
-                used_by="Adult",
-                slot="reason_for_happiness",
-                metaslot="any_of[range]",
-                used="EmploymentEvent",
-                inferred=False,
-            ),
-        ] == u["EmploymentEvent"]
-
-    # test methods also work for attributes
+    assert (
```

Comment:
the following assertions were indented and were thus running for every key, value pair in the view.usage_index.